src/ansys/fluent/core/solver/flobject.py

```diff
@@ -419,7 +421,7 @@ def _is_deprecated(self) -> bool:
 
     def is_active(self) -> bool:
         """Whether the object is active."""
-        attr = self.get_attr(_InlineConstants.is_active) and not self._is_deprecated()
+        attr = self.get_attr(_InlineConstants.is_active)
```

Comment:
User won't normally find deprecated objects in the settings API, so it is OK to leave them active.

Only the active children can be accessed from parent, so the deprecated objects should be kept active for backward compatibility of journals.

Reply:
sounds right, Thanks.